dashboard/src/app/workspaces/workspace-status/workspace-status.html

```diff
@@ -1,4 +1,4 @@
 <div class="workspace-status" flex layout="row" layout-align="start center">
   <workspace-status-indicator che-status="status"></workspace-status-indicator>
-  <span class="workspace-status-text">{{status | lowercase}}</span>
+  <span class="workspace-status-text" id="debug-id-workspace-status">{{status | lowercase}}</span>
```

Comment:
why do we need to call the id with "debug-id" prefix ?

Reply:
You are right - there ought to be added debug-id attribute - not id one.

Reply:
Yes, you are right. I will remove "debug-id" prefix from id name.